### ISSUE TITLE
Adding apps to jx creates different revisions in Helm instead of different releases

### ISSUE THREAD

**Issue by dgozalo** [jenkins-x/jx#3543]
### Summary 

Something weird is happening when installing apps with `jx add app <app-name> --namespace <ns>`.

When you install the same app in different namespaces, it's showing correctly in `jx` but when checking Helm list or status, there's only one release for the added app with as many revisions as the times the app has been installed in `different` namespaces. Also the install namespace listed in Helm is always the one where the app was first added.

### Steps to reproduce the behavior

1. Install the same app in different namespaces.
2. List them with `jx get apps` and check there are two rows for the same app in different namespaces.
3. Check `helm list` and see there is only one release for the <app-name> with as many revisions as times it has been added to jx, and `helm status <app-name>` to see that there's only one pod running.

### Expected behavior

Really need to check what we need this to be like, but I would expect that the app appears as different releases in Helm, with different pods and status so it can be queried for and truly be differentiated.

### Actual behavior

`jx add app jx-app-jacoco --namespace jx` 
`jx add app jx-app-jacoco --namespace jx2` 

Check the app correctly appears as installed in two namespaces when executing `jx get apps jx-app-jacoco`

The application correctly appears as installed in both namespaces:

```
Name          Version Chart Repository                Namespace Status   Description
jx-app-jacoco 0.0.123 http://chartmuseum.jenkins-x.io jx        DEPLOYED A Helm chart for the JenkinsX JaCoCo App
jx-app-jacoco 0.0.123 http://chartmuseum.jenkins-x.io jx2       DEPLOYED A Helm chart for the JenkinsX JaCoCo App
```
Check Helm for installed releases with `helm list`, and notice that the jx-app-jacoco release have been installed only once in the first namespace, but it has two revisions.

```
NAME              	REVISION	UPDATED                 	STATUS  	CHART                      	APP VERSION	NAMESPACE
jx-app-jacoco     	2       	Thu Apr  4 17:55:37 2019	DEPLOYED	jx-app-jacoco-0.0.123      	           	jx
``` 

And when querying for its state to Helm, check that there's only one pod running:

```
LAST DEPLOYED: Thu Apr  4 17:55:37 2019
NAMESPACE: jx
STATUS: DEPLOYED

RESOURCES:
==> v1/App
NAME                         AGE
jx-app-jacoco-jx-app-jacoco  9m9s

==> v1/Pod(related)
NAME                                          READY  STATUS   RESTARTS  AGE
jx-app-jacoco-jx-app-jacoco-76d66d5957-km9g2  1/1    Running  0         9m9s

==> v1/Role
NAME                         AGE
jx-app-jacoco-jx-app-jacoco  9m9s

==> v1/RoleBinding
NAME                         AGE
jx-app-jacoco-jx-app-jacoco  9m9s

==> v1/ServiceAccount
NAME                         SECRETS  AGE
jx-app-jacoco-jx-app-jacoco  1        9m9s

==> v1beta1/Deployment
NAME                         READY  UP-TO-DATE  AVAILABLE  AGE
jx-app-jacoco-jx-app-jacoco  1/1    1           1          9m9s

```

### Jx version

The output of jx version is:
(From Master branch)
```
jx                 1.3.980-dev+372af5b69
```

### Jenkins type

- [ ] Classic Jenkins
- [ X] Serverless Jenkins

### Kubernetes cluster

GKE, created using the `jx create cluster gke`

**Comment by pmuir** [comment]
Ironically this works fine with `--no-tiller` which is what I'm running.

**Comment by dgozalo** [comment]
Fixed by #3579
